native_client/python/Makefile

```diff
@@ -9,6 +9,7 @@ bindings-clean:
 # Enforce PATH here because swig calls from build_ext looses track of some
 # variables over several runs
 bindings-build: ds-swig
+	chmod -R 755 $$(find /usr/lib/python3.*/ -type f -name "*.py" | grep sysconfigdata_)
```

Comment:
@wasertech here is where I tried to change the permissions but unfortunately, the pipeline is still complaining

Reply:
<img width="749" alt="Screen Shot 2022-07-29 at 3 17 30 PM" src="https://user-images.githubusercontent.com/81978066/181837127-42f69264-975a-4acf-80ee-d97ad65b728b.png">

Reply:
Check how it is used, you should not never ever change its permissions. The error you get sounds like it's trying to be executed, which is NOT what you want.